lib/shared/models/selection-state.js

```diff
@@ -48,13 +48,41 @@ exports.selectDrive = (driveDevice) => {
  * selectionState.toggleDrive('/dev/disk2');
  */
 exports.toggleDrive = (driveDevice) => {
-  if (exports.isCurrentDrive(driveDevice)) {
-    exports.deselectDrive()
+  if (exports.isDriveSelected(driveDevice)) {
+    exports.deselectDrive(driveDevice)
   } else {
     exports.selectDrive(driveDevice)
   }
 }
 
+/**
+ * @summary Deselect all other drives and keep the current drive's status
+ * @function
+ * @public
+ * @deprecated
+ *
+ * @description
+ * This is a temporary function during the transition to multi-writes,
+ * remove this and its uses when multi-selection should become user-facing.
+ *
+ * @param {String} device - drive device identifier
```

Comment:
Sorry to be a pain, but here you've got `@param {String} device - drive device identifier` and on e.g. line 406 you've got `@param {String} driveDevice - drive device identifier` - perhaps it makes sense to choose one or the other and be consistent?

Reply:
No worries, you're right, I made that change to leverage object shorthand syntax but now that I've ended up not using it, it can be `driveDevice` with no drawbacks. It's done now.